examples/magic_image_arithmetic.py

```diff
@@ -0,0 +1,33 @@
+"""Basic example of using magicgui to create an Image Arithmetic GUI in napari."""
```

Comment:
I think including the arithmetic and parameter sweep examples in the napari docs makes them much more discoverable to new users. I'm +1 on that!